lib/util.js

```diff
@@ -387,12 +388,14 @@ exports.getAuthor = (body) => {
   }
   ownerinfo = new Entities().decode(ownerinfo);
   const channelMatch = ownerinfo.match(authorRegexp);
-  const userMatch = ownerinfo.match(aliasRegExp);
+  const userMatch = ownerinfo.match(aliasRegexp);
+  const verifiedMatch = ownerinfo.match(verifiedRegexp);
   return {
     id: channelMatch[1],
     name: channelMatch[2],
     avatar: url.resolve(VIDEO_URL, exports.between(ownerinfo,
       'data-thumb="', '"')),
+    verified: verifiedMatch && verifiedMatch[1] ? true : false,
```

Comment:
Since this ternary expression gives `true/false`, we can do `verifiedMatch && !!verifiedMatch[1]`

Reply:
Wow, I've never used something like that (a couple exclamation mark), I'll make that change now.